fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientReadPlatformServiceImpl.java

```diff
@@ -370,77 +305,141 @@ public Collection<ClientData> retrieveActiveClientMembersOfGroup(final Long grou
-            sqlBuilder.append("sbu.username as submittedByUsername, ");
-            sqlBuilder.append("sbu.firstname as submittedByFirstname, ");
-            sqlBuilder.append("sbu.lastname as submittedByLastname, ");
+            builder.append("c.transfer_to_office_id as transferToOfficeId, transferToOffice.name as transferToOfficeName, ");
+            builder.append("c.firstname as firstname, c.middlename as middlename, c.lastname as lastname, ");
+            builder.append("c.fullname as fullname, c.display_name as displayName, ");
+            builder.append("c.mobile_no as mobileNo, ");
+            builder.append("c.is_staff as isStaff, ");
+            builder.append("c.email_address as emailAddress, ");
+            builder.append("c.date_of_birth as dateOfBirth, ");
+            builder.append("c.gender_cv_id as genderId, ");
+            builder.append("cv.code_value as genderValue, ");
+            builder.append("c.client_type_cv_id as clienttypeId, ");
+            builder.append("cvclienttype.code_value as clienttypeValue, ");
+            builder.append("c.client_classification_cv_id as classificationId, ");
+            builder.append("cvclassification.code_value as classificationValue, ");
+            builder.append("c.legal_form_enum as legalFormEnum, ");
 
-            sqlBuilder.append("c.closedon_date as closedOnDate, ");
-            sqlBuilder.append("clu.username as closedByUsername, ");
-            sqlBuilder.append("clu.firstname as closedByFirstname, ");
-            sqlBuilder.append("clu.lastname as closedByLastname, ");
+            builder.append("c.submittedon_date as submittedOnDate, ");
+            builder.append("sbu.username as submittedByUsername, ");
+            builder.append("sbu.firstname as submittedByFirstname, ");
+            builder.append("sbu.lastname as submittedByLastname, ");
 
-            sqlBuilder.append("acu.username as activatedByUsername, ");
-            sqlBuilder.append("acu.firstname as activatedByFirstname, ");
-            sqlBuilder.append("acu.lastname as activatedByLastname, ");
+            builder.append("c.closedon_date as closedOnDate, ");
+            builder.append("clu.username as closedByUsername, ");
+            builder.append("clu.firstname as closedByFirstname, ");
+            builder.append("clu.lastname as closedByLastname, ");
 
-            sqlBuilder.append("cnp.constitution_cv_id as constitutionId, ");
-            sqlBuilder.append("cvConstitution.code_value as constitutionValue, ");
-            sqlBuilder.append("cnp.incorp_no as incorpNo, ");
-            sqlBuilder.append("cnp.incorp_validity_till as incorpValidityTill, ");
-            sqlBuilder.append("cnp.main_business_line_cv_id as mainBusinessLineId, ");
-            sqlBuilder.append("cvMainBusinessLine.code_value as mainBusinessLineValue, ");
-            sqlBuilder.append("cnp.remarks as remarks ");
+            // builder.append("c.submittedon as submittedOnDate, ");
```

Comment:
I don't know why this was left here as a comment. Also as I wrote above, please move back everything to its original place.

Reply:
This commented line is part of the original code, I mean I did not commented out this line.

<img width="919" alt="Screenshot 2022-12-09 at 8 49 00" src="https://user-images.githubusercontent.com/44206706/206728300-c128ee07-b2c2-4767-acb7-9a9d674b14a4.png">

Reply:
In terms of what the diff shows, this wasn't part of the query. If this is due to reordering again, please undo it and I can check it properly.